packages/form-config/src/components/FilesList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,168 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
These snapshots could be made smaller, no?

Reply:
@moshthepitt It's not one snapshot, they are different snapshots...no we cannot make any smaller